webpack/ForemanTasks/Components/TasksTable/TasksTableActions.js

```diff
@@ -93,18 +95,25 @@ export const unselectRow = id => ({
   payload: id,
 });
 
-export const cancelSelected = selected => async dispatch => {
-  let notAllCancelleble = false;
+export const actionSelected = (actionType, selected) => async dispatch => {
+  let notAllActionable = false;
   const promises = selected.map(task => {
-    if (task.isCancelleble) return dispatch(cancelTask(task.id, task.name));
-    notAllCancelleble = true;
+    if (actionType === RESUME && task.isResumeble)
+      return dispatch(resumeTask(task.id));
+    else if (actionType === CANCEL && task.isCancelleble)
+      return dispatch(cancelTask(task.id));
+    notAllActionable = true;
     return null;
   });
-  if (notAllCancelleble)
+  if (notAllActionable)
```

Comment:
tasks table reloads even though none of the tasks are resumable/cancelable and no data has been changed